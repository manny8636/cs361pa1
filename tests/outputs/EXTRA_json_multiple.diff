--- outputs/EXTRA_json_multiple.txt	2022-02-15 14:30:57.871097674 -0500
+++ expected/EXTRA_json_multiple.txt	2021-01-25 15:27:24.000000000 -0500
@@ -0,0 +1,6 @@
+Key-value pairs:
+{
+KEYS[first] = 15
+KEYS[second] = value
+}
+Success!
