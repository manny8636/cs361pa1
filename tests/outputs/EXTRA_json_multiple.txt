Parsing ../data/json_multiple.txt failed
