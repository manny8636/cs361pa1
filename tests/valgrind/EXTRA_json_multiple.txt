==3062924== Memcheck, a memory error detector
==3062924== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3062924== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==3062924== Command: ../json -o ../data/json_multiple.txt
==3062924== 
Success!
==3062924== 
==3062924== HEAP SUMMARY:
==3062924==     in use at exit: 38 bytes in 1 blocks
==3062924==   total heap usage: 4 allocs, 3 frees, 16,894 bytes allocated
==3062924== 
==3062924== LEAK SUMMARY:
==3062924==    definitely lost: 38 bytes in 1 blocks
==3062924==    indirectly lost: 0 bytes in 0 blocks
==3062924==      possibly lost: 0 bytes in 0 blocks
==3062924==    still reachable: 0 bytes in 0 blocks
==3062924==         suppressed: 0 bytes in 0 blocks
==3062924== Rerun with --leak-check=full to see details of leaked memory
==3062924== 
==3062924== For lists of detected and suppressed errors, rerun with: -s
==3062924== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
