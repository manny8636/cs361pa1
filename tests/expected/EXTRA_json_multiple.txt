Key-value pairs:
{
KEYS[first] = 15
KEYS[second] = value
}
Success!
